vlc-android/src/org/videolan/vlc/util/Strings.kt

```diff
@@ -0,0 +1,70 @@
+/*****************************************************************************
+ * Strings.java
+ *
+ * Copyright © 2011-2014 VLC authors and VideoLAN
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston MA 02110-1301, USA.
+ */
+
+package org.videolan.vlc.util
+
+import org.videolan.vlc.BuildConfig
+import java.text.DecimalFormat
+
+private val TAG = "VLC/UiTools/Strings"
+object Strings {
```

Comment:
You could remove the Strings class, and functions top-level and extension functions.
Call from Java would remane the same (with a jvm annotation) and calls from Kotlin would be easier